prompts/configs.go

```diff
@@ -66,19 +67,36 @@ func PromptForTargetConfig(targetName string, target *workflow.Target, existingC
 		}
 	}
 
-	var sdkClassName string
+	var sdkClassName string = ""
 	if !isQuickstart && output.Generation.SDKClassName != "" {
 		sdkClassName = output.Generation.SDKClassName
 	}
-	configFields := []huh.Field{
-		huh.NewInput().
-			Title("Name your SDK:").
-			Placeholder("your users will access SDK methods with <sdk_name>.doThing()").
-			Inline(true).
-			Prompt(" ").
-			Value(&sdkClassName),
+	_, err := charm.NewForm(
+		huh.NewForm(
+			huh.NewGroup(
+				huh.NewInput().
+					Title("Name your SDK:").
+					Description("your users will access SDK methods with myCompanySDK.doThing()").
+					// TODO: Are there languages where the default should not be PascalCase? What about terraform?
+					// TODO: If they are using the sample petstore spec during quickstart we should default to `MyPetStoreSDK`
```

Comment:
I'm not sure if we should actually. I feel like MyCompanySDK really sells that we want them to replace this. If we do my petstore spec they may think we always provide this value from the openapi spec.